source/typescript.txt

```diff
@@ -0,0 +1,29 @@
+.. _typescript-center:
+
+==========
+TypeScript
+==========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+Introduction
+------------
+
+Welcome to the documentation site for official MongoDB TypeScript support.
+Since TypeScript builds on JavaScript, you can use the
```

Comment:
Question:
I personally haven't heard "builds on" as the relationship and it seems like an ambiguous one. Usually, I see "transpiles to" or "compiles to" since there's a conversion process that happens to turn TypeScript into JavaScript before you can run it on a JS engine.

Reply:
@ccho-mongodb FWIW, I thought through the same thing during my review but ended up being okay with it. That said, I'm also fine with changing it.